sysutils/snooze/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        leahneukirchen snooze 0.5 v
+github.tarball_from archive
+revision            0
+categories          sysutils
+platforms           darwin
+license             public-domain
+maintainers         nomaintainer
+description         run a command at a particular time
+
+long_description \
+    snooze is a tool for waiting until a particular time and then \
+    running a command. Together with a service supervision system such \
+    as runit, this can be used to replace cron(8).
+
+homepage            https://github.com/leahneukirchen/snooze
```

Comment:
Specifying the `homepage` isn't necessary as it's automatically generated by the `github` portgroup off of `github.setup`.